.env.sample

```
@@ -0,0 +1,7 @@
+# following keys are required
+
+#backend endpoint
+REACT_APP_SERVER_URL=localhost/repo/api/vi
```

Small typo, please fix.

```suggestion
REACT_APP_SERVER_URL=localhost/repo/api/v1
```

---

updated